### PR TITLE
Fix documentation missing entry.

### PR DESCRIPTION
**Pull request by arjclark**
Addresses documentation mistake from #1501 

@hjoliver - this should now be correct.

### CODE REVIEW

File: doc/siterc.tex [hunk 1]
```diff
@@ -638,13 +638,17 @@ \subsection{[test battery]}
 The minimum set of directives that must be supplied to the batch system on the
 site to initiate jobs for the tests.
 
-\subsubsection[{[[}cylc suite settings{]]}]{[cylc] $\rightarrow$ [UTC mode]}
+\subsection[{[[}cylc suite settings{]]}]{[cylc]}
+
+Default values for entries in the suite.rc.
```
Comment (hjoliver) [inline doc/siterc.tex]: *...for entries in the suite.rc [cylc] section.*? (there will be other global items that correspond to other sections of the suite.rc file - e.g. in my authentication branch).

**Comment by hjoliver** [comment]
@arjclark - all of the cylc config file documentation thus far uses section headings that correspond exactly to the file section names.  You've inserted a new heading "[[cylc suite settings]]" which isn't a config file section (but it has double square brackets, which indicate a second-level section), and "[UTC mode]" has brackets although it isn't a section.  We should probably keep the style consistent.

**Comment by arjclark** [comment]
> You've inserted a new heading "[[cylc suite settings]]" which isn't a config file section (but it has double square brackets, which indicate a second-level section), and "[UTC mode]" has brackets although it isn't a section. We should probably keep the style consistent.

Will correct this.

**Comment by arjclark** [comment]
@hjoliver - hopefully that's right now.

**Comment by hjoliver** [comment]
That's got it, thanks.

**Pull request merged**